distributed/tests/test_s3.py

```diff
@@ -249,8 +263,22 @@ def test_read_text(e, s, a, b):
     assert all(isinstance(v, Future) for v in text)
 
 
+@gen_cluster(timeout=60, executor=True)
+def test_read_text_blocksize(e, s, a, b):
+    b = yield _read_text(test_bucket_name+'/test/accounts*', lazy=True,
+                         blocksize=20, collection=True)
+    assert b.npartitions == sum(ceil(len(b) / 20) for b in files.values())
+
+
+@gen_cluster(timeout=60, executor=True)
+def test_read_text_compression(e, s, a, b):
+    b = yield _read_text('distributed-test/csv/gzip/', compression='gzip')
+    result = yield e.compute(b)._result()
+    assert result == [line for k in sorted(csv_files)
+                           for line in csv_files[k].decode().split('\n')]
+
+
 def test_read_text_sync(loop):
-    pytest.importorskip('dask.bag')
```

Comment:
I've now removed all checks around `dask.bag`, which is generally expected to be around.  I've left the dask.dataframe checks in case Pandas isn't present.